README.md

```diff
@@ -77,27 +77,110 @@ Status codes tell us if a request was successful. The main groups are:
 
 ### Exercise: Use Open Movie Database API to fetch Harry Potter films and make a poster gallery.
 
-1. Make an API call to OMDb movies (in `poster-gallery/script.js`) to get all the information they have on Harry Potter. Read the [documentation](http://omdbapi.com/) to try and work out the url you need to use. You should expect an array of 10 objects with a Title, Year, imdbID, Type, and Poster property e.g.
-
-```
-{
-Search: [
-    {
-      Title: "Harry Potter and the Deathly Hallows: Part 2",
-      Year: "2011",
-      imdbID: "tt1201607",
-      Type: "movie",
-      Poster: "https://m.media-amazon.com/images/M/MV5BMjIyZGU4YzUtNDkzYi00ZDRhLTljYzctYTMxMDQ4M2E0Y2YxXkEyXkFqcGdeQXVyNTIzOTk5ODM@._V1_SX300.jpg"
-    }
-    ...
+1. Make an API call to OMDb movies (in `poster-gallery/script.js`) to get all Harry Potter films.
```

Comment:
maybe change this to be 'the first task is to make an API call to OMDb movies to get all the harry potter information they have. To do this follow the steps below:

Reply:
i think we shouldnt write 'harry potter movies' or 'harry potter movies and games' cos it might make them think they have to specify that somewehre?

Reply:
Done 🐼